Zend/tests/repeated_include/anonymous_class.phpt

```diff
@@ -0,0 +1,64 @@
+--TEST--
+Repeated include must not increase memory - anonymous class
+--SKIPIF--
+<?php
+// TODO test should pass even without opcache, but fixes are needed
+// related bug tracker https://bugs.php.net/bug.php?id=76982
+if (!extension_loaded('Zend OPcache') || !opcache_get_status()['opcache_enabled']) {
```

Comment:
any idea why `opcache_get_status()` is false on MacOS when testing tracing JIT? When testing JIT, I belive opcache must be enabled, thus it should return an array IMHO

Reply:
It seems to me that we need to `-d opcache.enable_cli=1`:
https://github.com/php/php-src/blob/b9f4fb8aefaeb3802d6f79f6aad7029b63e488a2/.github/workflows/push.yml#L79-L82
cc @iluuu1994

Reply:
@cmb69 Yes, that should be there.